api-service/README.md

```diff
@@ -2,22 +2,44 @@
+  crowdworkers to answer questions.
+* `webapp-demo` - is the frontend used by `job-server-demo` and is the interface
+  crowdworkers are expected to land at.
+
 
 ## Concepts
 
 The crowdsourcing tool is backed by by a Spanner Database. The database consists
 of:
 
-* `questions` - Holds all questions. Some questions are for training crowdworkers (`training` questions), some for testing crowdworkers (`test` questions), and others - typically most - are for getting answers from crowdworkers and for which the 'correct' answers is not known (`toanswer` questions). When being used for simple crowdsourced labelling, there are only `toansawer` questions.
-* `client_jobs` - Meta data for a job given which is available to a client. Each job specifies a set of questions in the `question` table (by `client_jobs.question_group_id = questions.question_group_id`), and expects a number of answers per question which will end up living in the `answer table`.
-* `answers` - Holds all answers to questions submitted by crowdsourcing. Each row in this table is associated to a particular `client_job` (by `client_jobs.client_job_key = answers.client_job_key`) and question (by `answers.question_id = questions.question_id`). There may be many `answers` rows per `client_job_key` and `question_id`.
-
-When being used to support third-party crowd-sourcing apps, where `test` and `training` questions are relevant, there is also a fourth table:
-
-* `question_scoring` - Each possible answer for a question can have a record here that maps it to a 'score'. This is intended to provide the ability to have a flexible way to score a crowd-workers answers to questions.
+* `questions` - Holds all questions. Some questions are for training
+  crowdworkers (`training` questions), some for testing crowdworkers (`test`
+  questions), and others - typically most - are for getting answers from
+  crowdworkers and for which the 'correct' answers is not known (`toanswer`
+  questions). When being used for simple crowdsourced labelling, there are only
+  `toansawer` questions.
+* `client_jobs` - Meta data for a job given which is available to a client. Each
+  job specifies a set of questions in the `question` table (by
+  `client_jobs.question_group_id = questions.question_group_id`), and expects a
+  number of answers per question which will end up living in the `answer table`.
+* `answers` - Holds all answers to questions submitted by crowdsourcing. Each
+  row in this table is associated to a particular `client_job` (by
+  `client_jobs.client_job_key = answers.client_job_key`) and question (by
+  `answers.question_id = questions.question_id`). There may be many `answers`
+  rows per `client_job_key` and `question_id`.
+
+When being used to support third-party crowd-sourcing apps, where `test` and
+`training` questions are relevant, there is also a fourth table:
+
+* `question_scoring` - Each possible answer for a question can have a record
```

Comment:
Could you give a bit more detail on this? It's not clear without context what this means.